policy/modules/system/systemd.te

```diff
@@ -396,6 +396,7 @@ kernel_dgram_send(systemd_machined_t)
 # This is a bug, but need for now.
 kernel_read_unlabeled_state(systemd_machined_t)
 
+domain_read_all_domains_state(systemd_machined_t)
```

Comment:
Is this really needed? If yes, do we know why?

Reply:
It is not required for all domain state.systemd_machined_tl reads cgroup of the container systemd process running as unconfined_service_t.
The parent process is systemd-nspawn. It looks like confining nspawn would help to narrow the required permissions. Atm systemd-nspawn runs as unconfined_service_t, too.